vpc/service/auth.go

```diff
@@ -57,12 +57,34 @@ func (vpcService *titusVPCAgentServiceAuthFuncOverride) AuthFuncOverride(ctx con
 	if err != nil {
 		return ctx, errors.Wrap(err, "Cannot parse remote certificate")
 	}
-	_, err = cert.Verify(x509.VerifyOptions{
+	err = vpcService.validateCert(cert)
+	if err != nil {
+		return nil, errors.Wrap(err, "Unable to verify client cert")
```

Comment:
you can use fmt.errorf now we're on go1.13

Reply:
Hah gosh, I have trouble remembering what is best practice between wrap, `%w`, and fmt.error